src/plots/stackedAreaPlot.ts

```diff
@@ -114,6 +114,28 @@ export class StackedArea<X> extends Area<X> {
     return this;
   }
 
+  /**
+   * Gets the offset of the y value corresponding to an x value of a given dataset. This allows other plots to plot
+   * points corresponding to their stacked value in the graph.
+   * @param dataset The dataset from which to retrieve the y value offset
+   * @param x The x value corresponding to the y-value of interest.
+   */
+  public yOffset(dataset: Dataset, x: any): number {
+    const stackingResult = this._stackingResult();
+    if (stackingResult == null) {
+      return undefined;
+    }
+    const datasetStackingResult = stackingResult.get(dataset);
+    if (datasetStackingResult == null) {
+      return undefined;
+    }
+    const result = datasetStackingResult.get(`${x}`);
```

Comment:
Using string interpolation here looks a little awkward? Isn't this the same as `.get(x)` or for typing purposes you could also do the very standard `.get(String(x))`